src/SamplesApp/UITests.Shared/Windows_UI_Input/PointersTests/EventsSequences.xaml.cs

```diff
@@ -349,6 +349,12 @@ private void ValidateHyperlinkTest(object sender, RoutedEventArgs e)
 					result =
 						args.Click()
 						&& args.End();
+#elif __ANDROID__
+					// KNOWN ISSUE: https://github.com/unoplatform/uno/issues/9080
+					// On Android the pointer clicked event is not raised properly
+					result =
+						args.One(PointerEnteredEvent)
+						&& args.MaybeSome(PointerExitedEvent);
```

Comment:
```suggestion
						&& args.MaybeOne(PointerExitedEvent)
						&& args.End();
```

We should probably also have a dedicated issue if the `Click` event is not fired on `Hyperlink` on Android. It's not just a flaky test issue